test/e2e/data/infrastructure-vsphere-govmomi/main/multi-disk/data-disks-patch.yaml

```diff
@@ -9,8 +9,10 @@ spec:
       dataDisks:
       - name: "disk_1"
         sizeGiB: 10
```

Comment:
Can we reduce the disk sizes a bit here and below? Seems a bit excessive

Reply:
Sure

Reply:
Thx!